README.md

```diff
@@ -57,9 +58,22 @@ If you'd like to develop on and build the riddc book, you should:
 - Start a python virtual environment: 
   Mac/Unix: `python3 -m venv ./` 
   Windows: `python3 -m venv .\`
+- Activate the environment with `source ./bin/activate`(Mac/Linux) or 
+`source .\bin\activate` (Windows)
+
+## Install system dependencies
+- On Mac: download Homebrew and install with `brew bundle`
+- On Windows:
+  * first download HDF5 from source: `https://www.hdfgroup.org/downloads/hdf5/source-code/`
```

Comment:
They can also install it form the binary in the link:
https://support.hdfgroup.org/ftp/HDF5/current/bin/

So they dont have to build the project from scratch using CMake